packages/block-library/src/media-text/edit.js

```diff
@@ -187,6 +191,35 @@ function MediaTextEdit( { attributes, isSelected, setAttributes } ) {
 	const onVerticalAlignmentChange = ( alignment ) => {
 		setAttributes( { verticalAlignment: alignment } );
 	};
+
+	const { imageSizes } = useSelect( ( select ) => {
+		const { getSettings } = select( 'core/block-editor' );
+		return pick( getSettings(), [ 'imageSizes' ] );
+	} );
```

Comment:
Do we need the `pick` (and the destructuring on L195)? Can we not ditch it by rewriting this as

```suggestion
	const imageSizes = useSelect( ( select ) => {
		const settings = select( 'core/block-editor' ).getSettings();
		return settings?.imageSizes;
	} );
```